test/torch/tensors/test_tensor.py

```diff
@@ -42,3 +43,84 @@ def test_create_pointer(self):
         x = torch.Tensor([1, 2])
         x.create_pointer()
         x.create_pointer(location=self.james)
+
+    def test_explicit_garbage_collect_pointer(self):
```

Comment:
We should add a test for garbage collection of a pointer to a pointer

Reply:
FYI: @robert-wagner and @mari-linhares: I'm working on getting GC for pointer->pointer->tensor working. It's been a lot more trouble than I expected.

Reply:
(I also didn't work on the 25th/26th of Dec... shameful I know)

Reply:
What problems are you having with pointer to pointer?

Reply:
What problems are you having with pointer to pointer?